numpy/core/src/umath/ufunc_object.c

```diff
@@ -6287,6 +6290,383 @@ ufunc_at(PyUFuncObject *ufunc, PyObject *args)
 }
 
 
+typedef struct {
+    PyArrayMethod_StridedLoop *strided_loop;
+    PyArrayMethod_Context *context;
+    NpyAuxData *auxdata;
+    /* Should move to flags, but lets keep it bools for now: */
+    npy_bool requires_pyapi;
+    npy_bool no_floatingpoint_errors;
+    PyArrayMethod_Context _full_context;
+    PyArray_Descr *_descrs[];
+} ufunc_call_info;
+
+
+void
+free_ufunc_call_info(PyObject *self)
+{
+    ufunc_call_info *call_info = PyCapsule_GetPointer(
+            self, "numpy_1.24_ufunc_call_info");
```

Comment:
Would a searchable comment help here? I ask because of the explicit versioning.

Reply:
I don't mind if this is not updated on every release, so long that it still warns that it might be updated on any release?  Adding a comment, but not sure it is searchable...